lib/EnvoyAPI.js

```diff
@@ -45,7 +47,13 @@ class EnvoyAPI {
       throw new Error('No token supplied.');
     }
 
+    this.token = token;
     this.baseUrl = process.env.ENVOY_BASE_URL || 'https://app.envoy.com';
+    this.newUrl = 'https://api.envoy.com/v1';
+    // External OAuth login url
+    this.envoyAuthUrl = 'https://app.envoy.com/a/auth/v0/authorize?response_type=code';
```

Comment:
How are we planning to use this? Currently integrations payloads include an integrations scoped JWT and a user scoped JWT for making calls so most integrations will not need to call auth directly.

Style note, a url constant should not have parameters on it. The `?response_type=code` should be added in place when the url is used.